### PR TITLE
SIDE-2479: Update User Aptitude Documentation

### PR DESCRIPTION
**Pull request by epiccoleman**
This PR updates the User Aptitude documentation with the some additional information, most significantly the list of JWT claims and the public key which can be used to validate the signatures of User Aptitude JWTs.

### CODE REVIEW

**Comment by EddieAtOlive** [comment]
This looks like a good improvement over the current docs for sure. Thanks!

One thing that might be worth considering is what path we use for the JWT's public key. Looks like it's currently set to `oliveai.dev/static/user-aptitude-jwt-rsa.pub` and we will eventually expose this through the `auth.oliveai.com` domain. It might be worth making the path on oliveai.dev mimic the path we'll eventually use. So for example, if we plan to make it: `auth.oliveai.com/public-keys/olive-helps/user-aptitude/jwt-rsa.pub`, then maybe we'd want the current one to be: `oliveai.dev/static/public-keys/olive-helps/user-aptitude/jwt-rsa.pub`. It's not a big deal, but it would be nice to settle on a consistent path now if we can do that quickly.

**Review by DanielWagersOlive — APPROVED** [review]

File: docs/src/markdown-pages/aptitudes/user.createJWT.md [hunk 1]
```diff
@@ -2,5 +2,20 @@
...
 links_js: "jwt"
 ---
-Receive a [JWT](https://jwt.io) with the currently logged in Olive Helps user's email address in the `subject` claim, the Loop ID in the `audience` claim, and a 5-minute `expiration`.
+Receive a [JWT](https://jwt.io) identifying the currently logged in Olive Helps user.
 
+## JWT Claims
+The claims in the JWT are as follows:
+
+Standard Claims:
+`sub` (Subject): contains the unique User ID of the current Olive Helps user
```
Comment (BrettMugglinOlive) [inline docs/src/markdown-pages/aptitudes/user.createJWT.md:11]: Super nitpick, but should these sentences here have periods at the end of them?

**Review by BrettMugglinOlive — APPROVED** [review]

**Comment by EddieAtOlive** [comment]
it's fine if this is intentional, but I wanted to double check that you do in fact want user-aptitude repeated twice here:
`user-aptitude/user-aptitude-jwt-rsa.pub`. Were you going for `user-aptitude/jwt-rsa.pub` or did you want that repeated?

File: docs/src/markdown-pages/aptitudes/user.createJWT.md [hunk 2]
```diff
@@ -2,5 +2,20 @@
...
 links_js: "jwt"
 ---
-Receive a [JWT](https://jwt.io) with the currently logged in Olive Helps user's email address in the `subject` claim, the Loop ID in the `audience` claim, and a 5-minute `expiration`.
+Receive a [JWT](https://jwt.io) identifying the currently logged in Olive Helps user.
 
+## JWT Claims
+The claims in the JWT are as follows:
+
+Standard Claims:
+`sub` (Subject): contains the unique User ID of the current Olive Helps user.
```
Comment (richseviora) [inline docs/src/markdown-pages/aptitudes/user.createJWT.md:11]: Markdown treats single new lines as standard space so they'll run together. To present them in a list like this you'll probably want to format them as markdown lists, or maybe as a table.

**Review by richseviora — CHANGES_REQUESTED** [review]
Something you probably didn't catch due to the Gatsby crashing issue :)

File: docs/src/markdown-pages/aptitudes/user.createJWT.md (hunk 2 above)
Comment (epiccoleman) [reply docs/src/markdown-pages/aptitudes/user.createJWT.md]: Fixed.

**Review by richseviora — APPROVED** [review]

**Pull request merged**